src/utils_format_graphite.c

```diff
@@ -60,11 +60,11 @@ static int gr_format_values(char *ret, size_t ret_len, int ds_num,
   else if (rates != NULL)
     BUFFER_ADD("%f", rates[ds_num]);
   else if (ds->ds[ds_num].type == DS_TYPE_COUNTER)
-    BUFFER_ADD("%llu", vl->values[ds_num].counter);
+    BUFFER_ADD("%" PRIsz, (uint64_t)vl->values[ds_num].counter);
   else if (ds->ds[ds_num].type == DS_TYPE_DERIVE)
     BUFFER_ADD("%" PRIi64, vl->values[ds_num].derive);
   else if (ds->ds[ds_num].type == DS_TYPE_ABSOLUTE)
-    BUFFER_ADD("%" PRIu64, vl->values[ds_num].absolute);
+    BUFFER_ADD("%" PRIsz, vl->values[ds_num].absolute);
```

Comment:
Ditto here, you need `PRIu64` for `uint64_t`.

Reply:
Thanks, I didn't quite get where to use PRIu64 before. Fixed.